modules/custom/az_core/src/Plugin/migrate/process/ManualMigrationLookup.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace Drupal\az_core\Plugin\migrate\process;
+
+use Drupal\Core\Database\Database;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\MigrateSkipProcessException;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Process Plugin to handle content that was manually migrated.
+ *
+ * @MigrateProcessPlugin(
+ *   id = "az_manual_migration_lookup"
```

Comment:
```suggestion
 *   id = "az_manual_migration_lookup"
```

could just call this `source_db_table_value_lookup` and then add fields and table name to the config.

Reply:
would look like this in practice 

```
          - plugin: source_db_table_value_lookup_by_id
            select_fields: title
            select_table: node
            select_index: nid
            source: target_id
 ```

Reply:
Probably too general and complicated to be useful?